core/modules/czar/bin/startCzar.py

```diff
@@ -98,6 +98,7 @@ def main():
     config.printTo(sys.stdout)
 
     # Configure logging
+    newlog.MDCPutPid()
```

Comment:
Hi Andy,

My bad, the branch to review is in lsst/log repository, I'll remove this branch in Qserv code.